crates/sui-core/src/node_sync/node_follower.rs

```diff
@@ -609,16 +619,20 @@ where
         aggregator: Arc<AuthorityAggregator<A>>,
         req: &DownloadRequest,
         node_sync_store: Arc<NodeSyncStore>,
+        metrics: GossipMetrics,
     ) -> SuiResult {
         let (cert, effects) = match req {
             DownloadRequest::Node(digests) => {
-                aggregator
+                metrics.total_cert_download_attempts.inc();
+                let res = aggregator
                     .handle_transaction_and_effects_info_request(
                         digests,
                         authorities.as_ref(),
                         None,
                     )
-                    .await?
+                    .await?;
+                metrics.total_cert_downloads.inc();
```

Comment:
The names of these metrics might lead a metrics user to think that they represent actual downloads (with requests / responses on the wire and all that) whereas these are requests that could lead to one download, or a check we have the info. Lets either use a clearer name, or measure actual downloads?

Reply:
the real metrics name is `gossip_total_cert_downloads` and I omit the prefix `gossip` to make it shorter.

I understand what you are saying but have a hard time coming up with a better name. I renamed them to `total_attempts_cert_downloads` and `gossip_total_attempts_cert_downloads` (so real metric name `gossip_total_attempts_cert_downloads` and `gossip_total_successful_attempts_cert_downloads`.

If they still don't make sense plz suggest one and I will put up another PR to udpate